src/protocolsupport/protocol/typeremapper/tile/TileEntitySkullRemapper.java

```diff
@@ -3,32 +3,57 @@
 import java.util.List;
 import java.util.function.Consumer;
 
+import org.bukkit.Bukkit;
 import org.bukkit.Material;
 import org.bukkit.block.data.BlockData;
 import org.bukkit.block.data.Rotatable;
 
+import protocolsupport.ProtocolSupport;
 import protocolsupport.api.MaterialAPI;
+import protocolsupport.api.ProtocolSupportAPI;
+import protocolsupport.api.ProtocolType;
+import protocolsupport.api.ProtocolVersion;
 import protocolsupport.protocol.typeremapper.legacy.LegacyBlockFace;
 import protocolsupport.protocol.typeremapper.tile.TileEntityRemapper.TileEntityWithBlockDataNBTRemapper;
 import protocolsupport.protocol.utils.types.nbt.NBTByte;
 import protocolsupport.protocol.utils.types.nbt.NBTCompound;
+import protocolsupport.protocol.utils.types.nbt.NBTFloat;
 import protocolsupport.utils.CollectionsUtils.ArrayMap.Entry;
 
 public class TileEntitySkullRemapper extends TileEntityWithBlockDataNBTRemapper {
-
+	protected boolean isPE;
+	public TileEntitySkullRemapper(boolean isPE)
+	{
+		this.isPE = isPE;
+	}
 	protected static byte getLegacyData(BlockData skull) {
 		if (skull instanceof Rotatable) {
 			return LegacyBlockFace.getLegacyRotatableId(((Rotatable) skull).getRotation());
 		}
 		return 0;
 	}
 
+	protected void register(List<Entry<Consumer<NBTCompound>>> list, Material skull, int skulltype, boolean isPE) {
```

Comment:
Wtf is all of that. You copied the function, why you were supposed to change and adapt existing one.

Reply:
I guess I committed an older version since I'm 100% sure I put it in the original method after testing it separately. I'll fix when I get home

Reply:
Leave BMan alone! /s